src/schemapack/_internals/cli/main.py

```diff
@@ -117,3 +119,53 @@ def check_datapack(
         load_datapack(datapack)
 
     print_final_success("The provided document complies with the specs of a datapack.")
+
+
+@cli.command()
+def export_mermaid(
+    *,
+    schemapack: Annotated[
+        Path,
+        typer.Argument(
```

Comment:
In the other commands, paths to schemapacks and datapacks are always given by command line options not arguments. We should probably make this consistent.